.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📚 Documentation
```

Comment:
Can we somehow order them to be:

- Feature Request
- Documentation
- StackOverflow
- Ask the Community
- Bug Report

If so that'd be best so that at least a users eyes have to consider the other options before opening a bug report :)

Reply:
(I would probably put feature request second to last, then :)

Reply:
I can't find documentation for ordering contact_links and actual templates. I switched Feature Request / Bug Report though.